build.gradle

```diff
@@ -57,8 +54,8 @@ sourceSets {
 }
 
 dependencies {
-    shadow gradleApi()
-    shadow localGroovy()
+    implementation gradleApi()
+    implementation localGroovy()
```

Comment:
localGroovy() should be in compileOnly, as Groovy is provided by Gradle as the plugin is applied.